I'm sorry, but an uncaught exception occurred.

While running game code:
<<<<<<< HEAD
  File "game/script.rpy", line 590, in script
    show eri ar right
Exception: Image 'eri' does not accept attributes 'ar right'.
=======
  File "game/script.rpy", line 1000, in script
    queue sound_effect_delayed(terror_sfx, 2.0)
  File "renpy/common/000statements.rpy", line 219, in execute_queue_music
    renpy.music.queue(
Exception: Audio channel 'sound_effect_delayed' is unknown.
>>>>>>> main

-- Full Traceback ------------------------------------------------------------

Full traceback:
<<<<<<< HEAD
  File "game/script.rpy", line 590, in script
    show eri ar right
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 1113, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 1076, in show_imspec
    renpy.config.show(name,
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\exports.py", line 739, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 246, in _duplicate
    args.extraneous()
  File "C:\Users\Greshnick\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'eri' does not accept attributes 'ar right'.
=======
  File "game/script.rpy", line 1000, in script
    queue sound_effect_delayed(terror_sfx, 2.0)
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\ast.py", line 1968, in execute
    self.call("execute")
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\ast.py", line 1950, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\statements.py", line 349, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 219, in execute_queue_music
    renpy.music.queue(
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\audio\music.py", line 207, in queue
    set_pause(False, channel=channel)
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\audio\music.py", line 545, in set_pause
    c = renpy.audio.audio.get_channel(channel)
  File "C:\University Documents\Lab\TM\renpy-8.2.0-sdk\renpy\audio\audio.py", line 928, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'sound_effect_delayed' is unknown.
>>>>>>> main

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012702
VisualNovelRenPy 1.0
<<<<<<< HEAD
Sat Feb 17 04:41:01 2024
=======
Sun Feb 18 17:13:01 2024
>>>>>>> main
